protected-api-access-protected-api/app.js

```diff
@@ -0,0 +1,89 @@
+/*
+ This application demonstrates how to issue a call to a protected API
+ using the On-behalf-of flow.  A request will be issued to the
+ Microsoft Graph /me endpoint on behalf of the authenticated user.
+*/
+
+// Microsoft Authentication Library (MSAL) for Node.js
+const msal = require('@azure/msal-node')
+
+// Node.js Express Framework
+const express = require('express')
+
+// Used to validate authentication tokens
+const jwt = require('express-jwt')
+const jwks = require('jwks-rsa')
+const jwtAuthz = require('express-jwt-authz')
+
+// Used to make the HTTP GET request to the Graph API
+const https = require('https')
+
+// MSAL configs
+const config = {
+  auth: {
+    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    clientId: '',
+   
+    // Client secret 'Value' (not the ID) from 'Client secrets' in app registration in Azure portal
+    clientSecret: '',
+   
+    // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+    authority: ''
+  }
+}
+
+
+// Initialize MSAL
+const msalConfidentialClientApp = new msal.ConfidentialClientApplication(config)
+
+// Initialize Express
+const app = express()
+
+// Validate the token from the user
+app.use(jwt({
+  secret: jwks.expressJwtSecret({
+    // Full URL, in the form of: https://login.microsoftonline.com/<tenant>/discovery/v2.0/keys
+    jwksUri: ''
+  }),
+ 
+  // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+  audience: config.auth.clientId,
+ 
+  // Full URL, in the form of: https://login.microsoftonline.com/<tenant>/v2.0
+  issuer: config.auth.authority + '/v2.0',
+ 
+  // Algorithm for MSAL tokens is R256
```

Comment:
Since you have them captured in a config object now.
```suggestion
  }),
  audience: config.auth.clientId,
  issuer: config.auth.authority + '/v2.0',

  // Algorithm for MSAL tokens is R256
```

Reply:
Thanks - Wasn't sure if the additional explanation was helpful or not